adafruit_wm8960.py

```diff
@@ -21,48 +25,1587 @@
 # * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
 """
 
-from adafruit_bus_device import i2c_device
-
-import time
+from busio import I2C
+from adafruit_bus_device.i2c_device import I2CDevice
 
 __version__ = "0.0.0+auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_WM8960.git"
 
+# I2C address (7-bit format for Wire library)
```

Comment:
For any constants below that the user of this library doesn't need, prepend a `_` to the name and wrap the value in `const()`. This will allow mpy-cross to strip the name out of the file and save space.

https://docs.circuitpython.org/en/latest/docs/design_guide.html#use-of-micropython-const

Reply:
Great idea. What are your thoughts on the prepended `WM8960_...` for user constants? For instance, rather than `adafruit_wm8960.WM8960_MIC_BOOST_GAIN_0DB`, would you prefer `adafruit_wm8960.MIC_BOOST_GAIN_0DB`? The only issue I see is some confusion if the user directly imports the constant, ie `from adafruit_wm8960 import MIC_BOOST_GAIN_0DB`.

Reply:
Nevermind, I see the convention clearly laid out here. https://docs.circuitpython.org/en/latest/docs/design_guide.html#driver-constant-naming